mkosi.images/build/mkosi.conf.d/opensuse/mkosi.conf

```diff
@@ -8,7 +8,7 @@ Environment=
         GIT_URL=https://github.com/bmwiedemann/openSUSE
         GIT_SUBDIR=packages/s/systemd
         GIT_BRANCH=master
-        GIT_COMMIT=1477f2646dd26ee3b166f26bd380a173fb649939
+        GIT_COMMIT=ace2a895124b90b17df404c5642cb985c837a194
```

Comment:
This change is not really bumping the version of the systemd package, you only have to check the history under https://github.com/bmwiedemann/openSUSE/commits/master/packages/s/systemd

Reply:
yeah there's no diff for the package, so this commit can be dropped

Reply:
@aafeijoo-suse Can you fix the `tools/fetch-distro.py` script to accomodate for your changes to use the opensuse github mirror?

Reply:
> @aafeijoo-suse Can you fix the `tools/fetch-distro.py` script to accomodate for your changes to use the opensuse github mirror?

Ok, I'll take a look later.